backend/src/models/MedicationCategory.js

```diff
@@ -17,35 +17,41 @@ import {
   DROPS_ICON,
   PATCHES_ICON,
 } from '../constants/MedicationCategory/medsIcons';
-
+import { MEDICATION_CATEGORIES } from '../constants/MedicationCategory/MedicationCategories';
 const COLORS = [PILLS_COLOR, SYRUP_COLOR, INHALER_COLOR, INJECTION_COLOR, DROPS_COLOR, PATCHES_COLOR];
 const ICONS = [PILLS_ICON, SYRUP_ICON, INHALER_ICON, INJECTION_ICON, DROPS_ICON, PATCHES_ICON];
 
 const { Schema } = mongoose;
 
 const medicationCategorySchema = new Schema({
-  name: String,
-  unit: { type: String, enum: [VOLUME_UNIT, QUANTITY_UNIT], default: QUANTITY_UNIT },
+  name: { type: String, enum: [...MEDICATION_CATEGORIES], default: MEDICATION_CATEGORIES[0], required: true },
```

Comment:
MEDICATION_CATEGORIES[0] is dangerous 
if someone is not aware of how to add new names, we may end up with different values here 

check out frontend/src/constants/picklistValues.js and frontend/src/constants/medTypes.js

Reply:
Medication constants moved to folder shared in root: shared/constants/MedicationCategory